ddtrace/ext/aws.py

```diff
@@ -9,23 +11,35 @@
     from ddtrace.span import Span
 
 
-BLACKLIST_ENDPOINT = ["kms", "sts"]
-BLACKLIST_ENDPOINT_TAGS = {
-    "s3": ["params.Body"],
+EXCLUDED_ENDPOINT = {"kms", "sts"}
+EXCLUDED_ENDPOINT_TAGS = {
+    "s3": {"params.Body"},
 }
 
 
-def _flatten_dict(d, sep=".", prefix=""):
-    # type: (Dict[str, Any], str, str) -> Dict[str, Any]
+def _flatten_dict(
```

Comment:
This is another helper which should probably exist in `trace_utils` since we do the same or similar in other integrations I think.